src/commands/poll.js

```diff
@@ -46,6 +47,19 @@ const EMOJI = [
  * @returns {Function} callback
  */
 exports.run = (client, message, args, callback) => {
+    let options = parseOptions(args, {
+        boolean: [
+            'sendchannel',
+            'extendable',
+        ],
+        alias: {
+            'sendchannel': 's',
+            'extendable': 'e',
+        }
+    });
+
+    args = options._;
```

Comment:
Function parameters [shouldn't be re-bound](https://github.com/NullDev/CSC-Bot/blob/master/.eslintrc#L110)